src/assets/foot_restraints/steel_ankle_cuffs/graphics.json

```diff
@@ -0,0 +1,644 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+{
+	"layers": [
+		{
```

Comment:
The layering is not suitable for the size of the object in the images, leading to weird situations like this:
![image](https://github.com/user-attachments/assets/d92542be-90c8-4aa8-80d7-0aeb757589e9)

You can solve this by switching this layer with the next one so that the images of bar and chain is sandwiched between the above and below body layers with the cuffs.